examples/todo_example/common/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export interface Todo {
+  completed: boolean;
+  id: string;
+  order: number;
+  priority: 'High' | 'Medium' | 'Low';
```

Comment:
Nit: Use an `enum` or constant instead of `'High'` throughout the code

Reply:
Also, this can be inferred from the `schema` defined below (if we separate it into a new file). It would look something like this:

server/saved_objects/schema.ts
```ts
export const TODO_SCHEMA = schema.object({
  completed: schema.boolean(),
  order: schema.maybe(schema.number()),
  priority: schema.oneOf([schema.literal('High'), schema.literal('Medium'), schema.literal('Low')]),
  title: schema.string(),
});

export type Todo = TypeOf<typeof TODO_SCHEMA>;
```

and then inside the routes:
```ts
// ...
validate: {
  body: TODO_SCHEMA
}
```

You could then use this instead of the interface defined here and it would make sure the types are kept in sync when the model changes.